From 01b0c1a8051f372e400d213ac125311c398e4b17 Mon Sep 17 00:00:00 2001
From: Raphael Kubo da Costa <rakuco@FreeBSD.org>
Date: Fri, 30 Dec 2011 23:27:31 -0200
Subject: [PATCH] Add missing include.

RefPtr::release() needs PassRefPtr.h for adoptRef(). The include is also
present in WebKit's wtf.

This should fix another build failure with clang.
---
 kjs/wtf/RefPtr.h |    1 +
 1 file changed, 1 insertion(+)

diff --git a/kjs/wtf/RefPtr.h b/kjs/wtf/RefPtr.h
index c16397e..493ab05 100644
--- a/kjs/wtf/RefPtr.h
+++ b/kjs/wtf/RefPtr.h
@@ -24,6 +24,7 @@
 
 #include <algorithm>
 #include "AlwaysInline.h"
+#include "PassRefPtr.h"
 
 namespace WTF {
 
-- 
1.7.10

